libs/api-interfaces/src/lib/detail-query-result.interface.ts

```diff
@@ -1,7 +1,8 @@
+import { WithTags } from './aggregate-views';
 import { ICommandFormAndLabels } from './commands';
 import { IViewModel } from './view-model.interface';
 
 export interface IDetailQueryResult<UViewModel extends IViewModel = IViewModel> {
```

Comment:
Can we not put the tags in `IViewModel` instead?

Reply:
Actually, the answer was to put them into a query result type specifically for categorizables (which are by definition tagable).